assets/minecraft/items/slime_ball.json

```
@@ -0,0 +1,117 @@
+{
+    "model": {
+        "type": "minecraft:condition",
+        "property": "minecraft:has_component",
+        "component": "minecraft:custom_name",
+        "on_true": {
+            "type": "minecraft:select",
+            "property": "minecraft:component",
+            "component": "minecraft:custom_name",
+            "cases": [
+                {
+                    "when": "PSC - Document",
+                    "model": {
+                        "type": "minecraft:model",
+                        "model": "minecraft:item/court/psc_document"
+                    }
+                }
+            ], [

```

```suggestion
                },
```
Do this for all continuing lines aswell